app/model/application.js

```diff
@@ -204,130 +181,15 @@ ApplicationSchema.methods.pushLabel = function() {
       color: '3A416F',
     },
   })
-  .catch(error => {
-    if (error.status === 422) { // Already created labels don't create errors
-      return Promise.resolve();
-    }
-    return Promise.reject(error);
-  });
-}
-
-// Fetches appHub file in repository
-// Returns promise of unsaved application object
-ApplicationSchema.methods.fetchApphub = function() {
-  let application = this;
-  const fullName = application.github.fullName;
-  app.log.silly(`application fetchApphub called for ${fullName}`);
-
-  return gh.request(`GET /repos/${fullName}/contents/.apphub`, {
-    token: application.github.APItoken,
-  })
-  .catch(error => {
-    if (error.status === 404) {
-      application.setProblem('missingApphub', true);
-      return Promise.resolve(application);
-    }
-    application.setProblem('missingApphub', false);
-    return Promise.reject(`Received ${error.status} from github`);
-  })
-  .then(apphubData => {
-    let apphubJSON = {};
-    try {
-      const apphubBase = new Buffer(apphubData.content, 'base64');
-      apphubJSON = JSON.parse(apphubBase.toString());
-    } catch (error) {
-      application.setProblem('unparsableApphub', true);
-      return Promise.resolve(application);
-    }
-    application.setProblem('unparsableApphub', false);
-
-    if (typeof apphubJSON.priceUSD === 'number') {
-      application.priceUSD = apphubJSON.priceUSD;
-      application.setProblem('invalidPrice', false);
-    } else if (typeof apphubJSON.priceUSD != 'undefined') {
-      application.setProblem('invalidPrice', true);
-    } else {
-      application.setProblem('invalidPrice', false);
-    }
-
-    if (typeof apphubJSON.issueLabel === 'string') {
-      application.github.label = apphubJSON.issueLabel;
-      application.setProblem('invalidLabel', false);
-    } else if (typeof apphubJSON.issueLabel != 'undefined') {
-      application.setProblem('invalidLabel', true);
-    } else {
-      application.setProblem('invalidLabel', false);
-    }
-
-    return application;
-  })
-}
-
-// Fetches .desktop file in repository
-// Returns promise of unsaved application object
-ApplicationSchema.methods.fetchDesktop = function() {
-  let application = this;
-  const repoName = application.github.name;
-  const fullName = application.github.fullName;
-  app.log.silly(`application fetchDesktop called for ${fullName}`);
-
-  return gh.request(`GET /repos/${fullName}/contents/data/${repoName}.desktop`, {
-    token: application.github.APItoken,
+  .then(data => {
+    return Promise.resolve(null);
```

Comment:
sounds good, was just curious to hear your thoughts. Might be worth adding a note about that to the style/coding guidelines ("no-op promises" or something)